src/transactions/transferFeedUtils.ts

```diff
@@ -435,28 +435,30 @@ function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTra
   const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
   const account = useMemo(
     () =>
-      fcTransferDetails?.quote.transferType === TransferType.TransferOut
+      fcTransferDetails?.quote
```

Comment:
```suggestion
      fcTransferDetails?.fiatAccountId
```

Reply:
maybe I'm missing something but it seems more valuable to check if there's a fiatAccountId before iterating thru the cached fiat account uses, instead of checking if there's a quote, which seems unrelated to the task of this hook, which is to find the correct fiat account

Reply:
Good point, updated!